plugins/auth-backend/src/identity/IdentityClient.test.ts

```diff
@@ -0,0 +1,277 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { JWT, JSONWebKey } from 'jose';
+import { utc } from 'moment';
+import { rest } from 'msw';
+import { setupServer } from 'msw/node';
+import {
+  getVoidLogger,
+  PluginEndpointDiscovery,
+} from '@backstage/backend-common';
+import { IdentityClient } from './IdentityClient';
+import { TokenFactory } from './TokenFactory';
+import { KeyStore, AnyJWK, StoredKey } from './types';
+
+const logger = getVoidLogger();
+
+class MemoryKeyStore implements KeyStore {
```

Comment:
Could actually move this and the existing one to a separate file and re-use instead